WireAuthentication/Sources/WireAuthenticationUI/Views/Root/RootViewModel.swift

```diff
@@ -24,7 +24,16 @@ import WireAuthenticationAPI
 @MainActor
 public final class RootViewModel: ObservableObject, Router {
 
+    enum ActiveSheet: Identifiable, Hashable {
+        var id: Self { self }
+
+        case authFlow
+        case noHistory(userID: UUID, cookies: [HTTPCookie])
+    }
+
     @Published var path = NavigationPath()
+    @Published var activeSheet: ActiveSheet? = .authFlow
+    @Published var showSSOFailureAlert: Bool = false
```

Comment:
suggestion: perhaps it would be nice to use the generic alert helper that @samwyndham made

Reply:
done